xarray/backends/api.py

```diff
@@ -135,6 +136,14 @@ def open_dataset(filename_or_obj, group=None, decode_cf=True,
         A variable or list of variables to exclude from being parsed from the
         dataset. This may be useful to drop variables with problems or
         inconsistent values.
+    only_variables: string or iterable, optional
+        A variable or list of variables to load from the dataset. This is
+        useful if you don't need all the variables in the file and don't want
+        to spend time loading them. Default is to load all variables.
+    format: string, optional
+        The format of the file to open (PyNIO engine only). This may be useful 
```

Comment:
I would simply say "currently only used by the PyNIO engine". We might eventually use this later, e.g., for selecting the fastest engine to use when open netCDF3 files (https://github.com/pydata/xarray/issues/887).